src/statsmodels/statsmodel.jl

```diff
@@ -86,6 +86,16 @@ function StatsBase.predict(mm::DataFrameRegressionModel, df::AbstractDataFrame;
     return(out)
 end
 
+# Predict function with confidence intervars
+function StatsBase.predict(mm::DataFrameRegressionModel, df::AbstractDataFrame, interval_type::Symbol, level::Real = 0.95; kwargs...)
+    newTerms = dropresponse!(mm.mf.terms)
+    mf = ModelFrame(newTerms, df; contrasts = mm.mf.contrasts)
+    newX = ModelMatrix(mf).m
+    yp = predict(mm, newX, interval_type, level; kwargs...) # this and the below two are different from the non-confint method above
+    out = NullableArray(eltype(yp), size(df, 1), 3)
```

Comment:
All vestiges of NullableArrays should be purged from the repo at some point, since the version of DataFrames that uses NullableArrays instead of DataArrays has been separated into https://github.com/JuliaData/DataTables.jl. So whatever uses DataArrays here is right.